source/administration/changelog.md

```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.24 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
 
+- **v5.24.2, release day TBD**
+  - Fixing an issue where ``ALTER PRIMARY KEY`` migration code does not work in Postgres 9.2. [MM-26514](https://mattermost.atlassian.net/browse/MM-26514)
```

Comment:
My GitHub PR title is technically incorrect. There's actually no such `ALTER PRIMARY KEY` sql query. :P 

I would just say "Fixed an issue where changing primary keys during migration would not work with Postgres versions lower than 9.3"